src/utilities.js

```diff
@@ -91,3 +93,18 @@ export function aOrAn(followingWord) {
         ? "an"
         : "a";
 }
+
+export function getRequiredCombatSkill(skillConstant) {
```

Comment:
I think keeping it as a method of the equipment class was fine since you could do: `this.equipment.obj.equipment.WEAPON.getRequiredCombatSkill()` but it doesn't really make much of a difference I guess

Reply:
Ehhhh I prefer utilities. Doesn't really need to be attached to equipment